README.md

```diff
@@ -91,6 +91,7 @@ The library is optimized for the following CPUs:
 * Intel Xeon CPU Max Series (formerly Sapphire Rapids HBM)
 * future Intel Xeon Scalable processors (code name Sierra Forest
 and Granite Rapids)
+* Arm Neoverse N1 and V1 processors
```

Comment:
I'd suggest we split the list by architecture for clarity.

* Intel 64/AMD64:
  * Intel Atom(R) processor (at least Intel SSE4.1 support is required)
  * ...
* AArch64:
  * Arm Neoverse N1 and V1 processors

Reply:
Sounds good

Reply:
Done